src/ui/utils/urlChecker.test.ts

```diff
@@ -1,6 +1,7 @@
 import { isValidHttpUrl } from "./urlChecker";
 
 const validUrl = [
+  "http://domain:1120/params/dasdadada?name=any",
   "https://a.long.sub-domain.example.com/foo/bar?foo=bar&boo=far#a%20b",
   "www.police.academy",
```

Comment:
To be honest, I'm not sure what KERI would do given just www for resolving. I don't have a KERIA server deployed at a www endpoint to know. But to a browser it would be a valid URL. I wouldn't be too strict here just in case. Worst case scenario it's just a pending connection that they can delete.

`https://a` is missing the `.com` or whatever, though maybe we shouldn't block that either @Sotatek-DukeVu - you might have `http://localhost` defaulting to port 80

Reply:
Yep. I supported this URL pattern in this PR. Please see below comment.

Reply:
hmm how is `http://localhost` supported but not `https://a`? i mean without the port where it defaults to 80

Reply:
@iFergal Yep. I supported this URL pattern in this PR. Please see below comment.
cc: @sdisalvo-crd

Reply:
Ah yes, for docker too, didn't even think about that one. All good, I thought `https://a` was still invalid based on the comments here.

Thanks!

Reply:
@iFergal 
I updated to support short pattern like `https://a` or `https://a?`